src/charts/ChartContainer.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Div = styled.div`
+  margin: 6rem auto;
+  width: 60rem;
```

Comment:
I recommend here adding `max-width: 60rem` and then `width:100%` to have it responsive (if the library supports responsive charts)